gulpfile.babel.js

```diff
@@ -292,83 +363,115 @@ gulp.task('rpm:linux', ['package:linux'], (done) => {
   });
 });
 
+gulp.task('rpm:linux:32', ['package:linux:32'], (done) => {
+  const redhat = require('electron-installer-redhat');
+
+  redhat(_.extend({}, redhatDefaults, {
+    src: `dist/${packageJSON.productName}-linux-ia32`,
+    arch: 'i386',
+  }), (err) => {
+    console.log('32bit rpm package built');
+    if (err) return done(err);
+    done();
+  });
+});
+
+gulp.task('rpm:linux:64', ['package:linux:64'], (done) => {
+  const redhat = require('electron-installer-redhat');
+
+  redhat(_.extend({}, redhatDefaults, {
+    src: `dist/${packageJSON.productName}-linux-x64`,
+    arch: 'amd64',
+  }), (err) => {
+    console.log('32bit rpm package built');
+    if (err) return done(err);
+    done();
+  });
+});
+
 gulp.task('make:linux', ['deb:linux', 'rpm:linux'], (done) => {
```

Comment:
Hmmmm, this line might not work in my opinion.

Gulp will fire off both the `deb:linux` and `rpm:linux` tasks simultaneously.  Both those tasks run 32bit/64bit rebuilds of native modules.  Depending on how quick each task runs a 32bit build might have been rebuilt into 64bit before the copy process.  We should use something like this https://www.npmjs.com/package/run-sequence (until gulp 4.0 is released) that will force execution order.